src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,10 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  width: calc((#{$card-width} * 4) + (#{$gap} * 3));
+}
+
+.catalog--position {
```

Comment:
Using BEM methodology, '--position' is not a good choice for a modifier name. A better choice could be '--centered' if the goal is to center the catalog. Keep in mind that BEM modifiers should represent state or behavior.